com.microsoft.mrtk.graphicstools.unity/Editor/Utilities/InspectorUtilities.cs

```diff
@@ -24,9 +121,7 @@ public static class InspectorUtilities
             // Register the creation in the undo system.
             Undo.RegisterCreatedObjectUndo(gameObject, "Create " + gameObject.name);
 
-            Selection.activeObject = gameObject;
-
-            return gameObject;
+            return SetupCanvas(gameObject, menuCommand);
```

Comment:
Not all objects `CreateGameObjectFromMenu` creates need to be under a canvas. For example a HoverLight, DistantLight etc: 
![image](https://user-images.githubusercontent.com/13305729/222251150-ecbba6d4-4ef6-4fb1-bdc9-6f5fea9526a4.png)

Maybe we should add another method (like `CreateCanvasGameObjectFromMenu`) for canvas specific objects? Or add a bool parameter to this method which will allow the calling function to decide if a canvas is needed or now?